modules/sharedaddy.php

```diff
@@ -20,4 +20,10 @@
 
 function sharedaddy_loaded() {
 	Jetpack::enable_module_configurable( __FILE__ );
+	add_filter( 'jetpack_module_configuration_url_sharedaddy', 'jetpack_sharedaddy_configuration_url' );
+}
+
+function jetpack_sharedaddy_configuration_url() {
+	$site_suffix  = Jetpack::build_raw_urls( get_home_url() );
+	return 'https://wordpress.com/sharing/buttons/' . $site_suffix;
```

Comment:
That works, but only once the site is connected to WordPress.com. I think we would need to add a check and link to Settings > Sharing when the site is in dev mode, or when the user visiting the page is not linked to WordPress.com yet.

Reply:
164ddc4ac6e8e295cc88a293bcc7439889452e2c